pages/index.tsx

```diff
@@ -116,15 +126,46 @@ export default function Home() {
                           <AiFillPlayCircle className="hover:text-red-400 transition ease-in-out md:text-5xl text-6xl" />
                         </button>
                       ) : (
-                        <button
-                          onClick={() => {
-                            startRecording();
-                          }}
-                        >
-                          <ImLoop2 className="py-2 hover:text-red-400 transition ease-in-out md:text-5xl text-6xl" />
-                        </button>
+                        <div className="flex items-center gap-2">
+                          <button
+                            onClick={() => {
+                              startRecording();
+                            }}
+                          >
+                            <ImLoop2 className="text-yellow-800 py-2 hover:text-red-400 transition ease-in-out md:text-5xl text-6xl" />
+                          </button>
+                          <button
+                            className="px-4 py-1.5 bg-amber-800 bg-opacity-30 hover:bg-opacity-50 duration-150 hover:duration-100 rounded-md"
+                            onClick={async () => {
+                              const file = (await blobToBase64(
+                                blob!
+                              )) as string;
+                              const formData = new FormData();
+                              formData.append("file", file);
+                              formData.append(
+                                "upload_preset",
+                                "unsigned_preset"
+                              );
+                              const result = await fetch(
+                                "https://api.cloudinary.com/v1_1/dxxmohqvs/auto/upload",
+                                {
+                                  method: "POST",
+                                  body: formData,
+                                }
+                              ).then((res) => res.json());
+                              console.log(result);
+
+                              setUploadedUrl(
+                                `https://res.cloudinary.com/dxxmohqvs/video/upload/${result.public_id}.mp4`
+                              );
+                            }}
+                          >
+                            ✨ create
+                          </button>
+                        </div>
                       )}
                     </div>
+                    {uploadedUrl}
```

Comment:
we're going to reformat this later right?

Reply:
yep!